src/Gradient.php

```diff
@@ -22,6 +22,9 @@ class Gradient
 	 */
 	private $colorConverter;
 
+	const TYPE_LINEAR = 2;
```

Comment:
Please, keep class constants declared first in the class. The order is constants, properties, constructor, other methods (preferrably public first, private second).